packages/synapse-react-client/src/lib/style/components/_feed-cards.scss

```diff
@@ -73,9 +73,9 @@
           border: 1px solid SRC.$primary-action-color;
           display: inline-flex;
           align-content: flex-start;
-          border-radius: SRC.$button-border-radius;
+          border-radius: 100px;
           padding: 5px 20px;
-          text-transform: SRC.$button-text-transform;
+          text-transform: uppercase;
```

Comment:
Revert styling of feed card categories, see "DATA RELEASE": 
<img width="1382" alt="feature_RSSFeedCards" src="https://user-images.githubusercontent.com/26949006/231906893-c405ba3a-e4e0-49f8-a228-e63d7b7dc286.png">